### PR TITLE
add hardware matrix

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+|--------------------------------|--------------------------------------------------|
+| **CPU**                        | Intel Core i5-12600K 3.7                         |
+|                                | Noctua NH-U12S CPU Cooler                        |
+| **GPU**                        | N/A                                              |
+| **RAM**                        | Slot 1: Kingston KF436C17BBK4/32                 |
+|                                | Slot 2: Kingston KF436C17BBK4/32                 |
+|                                | Slot 3: Kingston KF436C17BBK4/32                 |
+|                                | Slot 4: Kingston KF436C17BBK4/32                 |
+| **SSD**                        | Intel 670p 512 GB M26472-201 NVME                |
+| **Flash memory**               | TBD                                              |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/zir_blazer/hardware_matrix.md:18]: Information about flash memory should be added

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
```
Comment (sulewskiprzemyslaw) [inline docs/variants/zir_blazer/hardware_matrix.md:42]: Information about flash memory should be added

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+| **Network**                    | Local network wired connection                   |
+| **Attached devices**           | 1. SanDisk Ultra  Flair USB 3.0 16 GB            |
+|                                | 2. Kingston DataTraveler 3.1Gen1 16 GB           |
+|                                | 3. SanDisk Ultra microSD HC I 16 GB              |
+|                                | 4. Logitech, Inc. Keyboard K120                  |
+|                                | 5. Dell Mouse MS116p                             |
+|                                | 6. 1-port USB Type-C                             |
+| **Power Supply**               | SeaSonic FOCUS Plus Platinum                     |
+| **Power Control**              | sonOFF                                           |
+| **Remote Testing Environment** | TBD                                              |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/zir_blazer/hardware_matrix.md:28]: This section should be completed

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/zir_blazer/hardware_matrix.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: added

File: docs/variants/zir_blazer/hardware_matrix.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: added

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## LA1E012608
+
+| Component                      | Description                                      |
+|--------------------------------|--------------------------------------------------|
+| **CPU**                        | Intel Core i5-12600K 3.7                         |
+|                                | Noctua NH-U12S CPU Cooler                        |
+| **GPU**                        | N/A                                              |
```
[the remark this answers is not in the document]
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: removed

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+The tables below contain information about current components of
+workstations for Zir_Blazer available in 3mdeb testing laboratory.
+
+## LA1E312159
+
+| Component                      | Description                                      |
+|--------------------------------|--------------------------------------------------|
+| **CPU**                        | Intel Core i5-12600K 3.7                         |
+|                                | Noctua NH-U12S CPU Cooler                        |
+| **GPU**                        | N/A                                              |
```
[the remark this answers is not in the document]
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: removed

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+| **Flash memory**               | TBD                                              |
+| **Network**                    | Local network wired connection                   |
+| **Attached devices**           | 1. SanDisk Ultra  Flair USB 3.0 16 GB            |
+|                                | 2. Kingston DataTraveler 3.1Gen1 16 GB           |
+|                                | 3. SanDisk Ultra microSD HC I 16 GB              |
+|                                | 4. Logitech, Inc. Keyboard K120                  |
+|                                | 5. Dell Mouse MS116p                             |
+|                                | 6. 1-port USB Type-C                             |
+| **Power Supply**               | SeaSonic FOCUS Plus Platinum                     |
+| **Power Control**              | sonOFF                                           |
```
[the remark this answers is not in the document]
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: I deleted line, with this workstation we don't have set up power control

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
```
[the remark this answers is not in the document]
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: rename and add ip

File: docs/variants/zir_blazer/hardware_matrix.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+| **Remote Testing Environment** | TBD                                              |
```
[the remark this answers is not in the document]
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: I deleted line, with this workstation we don't have set up power control

File: docs/variants/zir_blazer/hardware_matrix.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/variants/zir_blazer/hardware_matrix.md]: added RTE version and ip

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**